Installation Graphics Update (Cleaned Up)

KillerDroid96 opened the pull request:
-Changed CrDroid Font to Slant
-Changed format of info displayed
-Added more information to be displayed 
-Changed ROM Version to crDroid Version (May revert this)
-Removed Kernel Version for now

KillerDroid96 committed:
Installation Graphics Update (Cleaned Up)

-Changed CrDroid Font to Slant
-Changed format of info displayed
-Added more information to be displayed
-Changed ROM Version to crDroid Version (May revert this)
-Removed Kernel Version for now

-To Add Kernel (Change Path):
kernel = ("/home/killerdroid96/Github/CRD/out/target/product/dumpling/obj/KERNEL_OBJ/include/config/kernel.release")
kernelver = open(kernel).read()
script.Print(" Kernel Version   : %s"%(kernelver));

adithya2306 commented:
Better than the previous one but I dont think we need maintainer info, as @neobuddy89 mentioned in the previous request.

KillerDroid96 committed:
Installation Graphics Update

-Changed CrDroid Font to Slant
-Changed format of info displayed
-Added more information to be displayed
-Changed ROM Version to crDroid Version (May revert this)
-Removed Kernel Version for now

gwolf2u commented:
not needed as of now
